lib/src/main/java/graphql/nadel/engine/transform/hydration/batch/NadelNewBatchHydrator.kt

```diff
@@ -497,6 +488,61 @@ internal class NadelNewBatchHydrator(
         }
     }
 
+    context(NadelBatchHydratorContext)
+    private fun getFilteredInstructionFromList(
```

Comment:
Can we rename this? You named this "filtered" instruction because you were filtering and returning a List before, but that isn't the case now.

Can we just call this `getHydrationInstructionForSourceInput` instead?

Reply:
good call, thanks for picking up on this. Changed